recipe/meta.yaml

```
@@ -1,19 +1,17 @@
 {% set name = "nionui-tool" %}
-{% set version = "0.4.22" %}
+{% set version = "5.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/nion-software/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: 50e2fa59a4ebba4e1fe313a091baaa0e688b4f65d19eb617c64fd605ce18bf5c
-  patches:
-    - patches/0001-Do-not-build-fat-binary.patch
+  sha256: 876f76592a95d7c6b9bfc58af325193ad59ebcecb50ab766b5254123c850e633
 
 build:
   number: 0
-  skip: true  # [py2k or python_impl == 'pypy']
+  skip: true  # [py<311 or py2k or python_impl == 'pypy']
```

`py2k` is subsumed by `py<311`. So we can drop it off 

Also conda-forge no longer builds for Python 2 or PyPy. So we can simplify this like so

```suggestion
  skip: true  # [py<311]
```

---

Looks like this suggestion was missed. Have converted into a new issue: https://github.com/conda-forge/nionui-tool-feedstock/issues/50